packages/elm/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # @webpack-blocks/elm - Changelog
 
+## 1.0.0-beta.2
+
+- Added `package-lock.json`
```

Comment:
Why is this in the change log? I mean why users should care about changes they will never see? ;-)

Reply:
Just wanted to provide a reason why there is a new version 😉

Reply:
Right answer: there shouldn’t be a new version because there are no changes that may affect users ;-)

Reply:
I was aiming for consistency. The `package-lock.json` makes sense and I would like to publish it for all core packages at once.

Reply:
It make sense for us but not for users, this shouldn’t be in the change log. It won’t be even published to npm.

Reply:
So what should we do to publish a new version, but without this line in the changelog? Should I put it in parenthesis? ^^

Reply:
But what’s the reason to publish a new version without any changes?

Reply:
Ahh, my bad. Sorry, I got something wrong about the `package-lock.json`.
For future reference: Here is its [documentation](https://docs.npmjs.com/files/package-lock.json).

In this case it doesn't make sense to bump the version, you are right 👍

Reply:
🍕